src/plugins/explore/public/application/index.tsx

```diff
@@ -28,6 +31,29 @@ interface ExploreRouteProps {
 type ExploreComponentProps = ExploreRouteProps &
   Partial<Pick<AppMountParameters, 'setHeaderActionMenu'>>;
 
+// Component that handles page context for all Explore flavors using osdUrlStateStorage
+const ExplorePageContextProvider: React.FC<{
+  flavor: ExploreFlavor;
+  children: React.ReactNode;
+}> = ({ flavor, children }) => {
```

Comment:
you are passing flavour but not using it

Reply:
have cleaned out explore